server/bin/service-key.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+"""Create an imaginary service account with a single API key.
+
+Create a User record and a matching API key in the database, which can be used
+as a service account. The API key provides authentication for services and the
+User record allows for ID to name translation.
+
+The primary purpose is to serve as a "legacy" collector for the 0.69-11
+passthrough server, but it could also be used for other API clients which don't
+require a specific SSO user identity.
+"""
+
+from argparse import ArgumentParser
+from datetime import datetime, timezone
+import os
+from pathlib import Path
+import sys
+import uuid
+
+import jwt
+
+from pbench.common.logger import get_pbench_logger
+from pbench.server import PbenchServerConfig
+from pbench.server.database import init_db
+from pbench.server.database.models.api_keys import APIKey
+from pbench.server.database.models.users import User
+
+
+def main(options, name):
+    """Create a Pbench Server service account"""
+    config = PbenchServerConfig.create(options.cfg_name)
+    logger = get_pbench_logger(name, config)
+    init_db(config, logger)
+
+    username = options.username
+
+    # You can specify a nonsense string or let the code generate a real (and
+    # "very likely unique") UUID.
+    id = options.identity if options.identity else str(uuid.uuid1())
```

Comment:
You like [`uuid1()`](https://docs.python.org/3/library/uuid.html#uuid.uuid1) better than [`uuid4()`](https://docs.python.org/3/library/uuid.html#uuid.uuid4)?  (Given that you're not passing any arguments, they probably do the same thing, so I would use `uuid4()`....)